modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -747,7 +747,8 @@ ONNXCompiled::ONNXCompiled(const gapi::onnx::detail::ParamDesc &pp)
                             in_tensor_info.end(),
                             [](const cv::gimpl::onnx::TensorInfo &p) {
                                 return p.type == ONNX_TENSOR_ELEMENT_DATA_TYPE_FLOAT
-                                    || p.type == ONNX_TENSOR_ELEMENT_DATA_TYPE_UINT8;
+                                    || p.type == ONNX_TENSOR_ELEMENT_DATA_TYPE_UINT8
+                                    || p.type == ONNX_TENSOR_ELEMENT_DATA_TYPE_INT32;
                             })
                 && "Only FP32 and U8 inputs for NN are supported");
```

Comment:
But exception message still only about `FP32` and `U8`.

Btw, all types supported in ONNXRT: https://github.com/microsoft/onnxruntime/blob/main/include/onnxruntime/core/session/onnxruntime_c_api.h#L177-L203

We also need to cover `_INT64`

Reply:
To cover `_INT64` we also need handling here: https://github.com/opencv/opencv/blob/762ac0f2f394cdbc2837be5ee0735d0812997156/modules/gapi/src/backends/onnx/gonnxbackend.cpp#L467-L474

Reply:
Note, that there is not `CV_I64` type, in such case we usually apply WA and upcast data from `CV_32` cv::Mat to `int64_t`. Here is example from OpenVINO backend: https://github.com/opencv/opencv/blob/4.x/modules/gapi/src/backends/ov/govbackend.cpp#L144-L146